rl-sideswipe-access/android/app/src/main/java/com/rlsideswipe/access/service/ScreenCaptureService.kt

```
@@ -0,0 +1,1446 @@
+                    points.addAll(templateMatches.map { match ->
+                        PredictionOverlayService.PredictionPoint(match.x, match.y, 0f)
+                    })
+                    points.add(PredictionOverlayService.PredictionPoint(screenWidth * 0.1f, screenHeight * 0.1f, 2f))
+                    points.add(PredictionOverlayService.PredictionPoint(screenWidth * 0.5f, screenHeight * 0.5f, 1f))
+                    points.add(PredictionOverlayService.PredictionPoint(screenWidth * 0.9f, screenHeight * 0.9f, 2f))
+                    points
+                }
+                
+                PredictionOverlayService.updatePredictions(overlayPoints)
+            } else {
+                val isTflite = try { com.rlsideswipe.access.BuildConfig.FLAVOR?.contains("tflite") == true } catch (e: Exception) { false }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)